FlowerApi/Controllers/CartController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using FlowerApi.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace FlowerApi.Controllers
```

Comment:
Gal name būtų geriau ne FlowerApi, o FlowerShopApi ar kažkas panašiai? Nes čia ne tik gėles tvarkom, bet ir userius,  cartus ir t.t. Aišku čia simple refactor in VS, tai all good 👍